### DISCUSSION TITLE
ph and protonation

### DISCUSSION THREAD

**Discussion by Nag-Nad** [rdkit/rdkit#6558] [Q&A]
Good day to The Community,

I have recently started to use rdkit for a project and I am not sure if I am on a right track. Regarding the function Chem.AddH(), I would like to know in which pH does rdkit protonate the smile molecules? Since it can solve my doubts and help me move on, I would appreciate every single help. 

Thank you.

**Comment by bp-kelley** [comment]
AddHs only fills in valences, there is no pH model being used besides this.
